src/recommendations/data/reducers.js

```diff
@@ -0,0 +1,27 @@
+import { USER_RECOMMENDATIONS } from './actions';
+
+export const defaultState = {
+  recommendations: [],
+  isLoading: true,
+  showError: false,
+};
+
+const reducer = (state = defaultState, action) => {
+  switch (action.type) {
+    case USER_RECOMMENDATIONS.SUCCESS:
+      return {
+        ...state,
+        recommendations: action.payload.recommendations,
```

Comment:
is the recommendations coming from Algolia according to the format that we need in our components?

Reply:
Yes I coordinated with Attiya and She was ok with the current algolia response